ote/src/clj/ote/integration/import/gtfs.clj

```diff
@@ -155,7 +159,7 @@
   (let [db (:db ote.main/ote)]
     (clojure.java.jdbc/execute! db ["TRUNCATE TABLE gtfs_package RESTART IDENTITY CASCADE"])
     (clojure.java.jdbc/execute! db ["INSERT INTO gtfs_package (id) VALUES (1)"])
-    (let [bytes (with-open [in (io/input-stream "hsl_gtfs.zip")]
+    (let [bytes (with-open [in (io/input-stream "/Users/tatuta/Downloads/gtfs_tampere (2).zip" #_"hsl_gtfs.zip")]
```

Comment:
Not sure but I think that our ec2 instance has different path for zip files.

Reply:
This is a function for REPL testing only and is commented out,